### ISSUE TITLE
Alot of warns

### ISSUE THREAD

**Issue by beerko100500** [asterisk-java/asterisk-java#259]
```
[Asterisk-Java AgiServer-0] INFO org.asteriskjava.fastagi.DefaultAgiServer - Listening on *:4573.
[main] INFO org.asteriskjava.manager.internal.ManagerConnectionImpl - Connecting to XXX.XXX.XXX.XXX:5038
[Asterisk-Java ManagerConnection-0-Reader-0] INFO org.asteriskjava.manager.internal.ManagerConnectionImpl - Connected via Asterisk Call Manager/2.10.4
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ManagerConnectionImpl - Unsupported protocol version 'Asterisk Call Manager/2.10.4'. Use at your own risk.
[main] INFO org.asteriskjava.manager.internal.ManagerConnectionImpl - Successfully logged in
[main] INFO org.asteriskjava.manager.internal.ManagerConnectionImpl - Determined Asterisk version: Asterisk 13
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sipshowregistry' to 'Show SIP registrations (text format).  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'agi' to 'Add an AGI command to execute by Async AGI.  (Priv: agi,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queuelog' to 'Adds custom entry in queue_log.  (Priv: agent,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dongleremove' to 'Remove a dongle. (Priv: system,config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dahdidialoffhook' to 'Dial over DAHDI channel while offhook.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgeunmute' to 'Unmute a Confbridge user.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'getconfigjson' to 'Retrieve configuration (JSON format).  (Priv: system,config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'bridgetechnologysuspend' to 'Suspend a bridging technology.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dialplanextensionremove' to 'Remove an extension from the dialplan  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'iaxregistry' to 'Show IAX registrations.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'messagesend' to 'Send an out of call message to an endpoint.  (Priv: message,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'aocmessage' to 'Generate an Advice of Charge message on a channel.  (Priv: aoc,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queuepenalty' to 'Set the penalty for a queue member.  (Priv: agent,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'waitevent' to 'Wait for an event to occur.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'agentlogoff' to 'Sets an agent as no longer logged in.  (Priv: agent,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'donglereload' to 'Reload a module configuration. (Priv: system,config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'controlplayback' to 'Control the playback of a file being played to a channel.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'extensionstatelist' to 'List the current known extension states.  (Priv: call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'muteaudio' to 'Mute an audio stream.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgekick' to 'Kick a Confbridge user.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sipshowpeer' to 'show SIP peer (text format).  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'presencestate' to 'Check Presence State  (Priv: call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgeunlock' to 'Unlock a Confbridge conference.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'bridgekick' to 'Kick a channel from a bridge.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'coreshowchannels' to 'List currently active channels.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sorcerymemorycacheexpire' to 'Expire (remove) ALL objects from a sorcery memory cache.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'bridgetechnologyunsuspend' to 'Unsuspend a bridging technology.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'logoff' to 'Logoff Manager.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'mixmonitormute' to 'Mute / unMute a Mixmonitor recording.  (Priv: system,call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgemute' to 'Mute a Confbridge user.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgelock' to 'Lock a Confbridge conference.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queuesummary' to 'Show queue summary.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'bridge' to 'Bridge two channels already in the PBX.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgelistrooms' to 'List active conferences.  (Priv: reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'skinnylines' to 'List SKINNY lines (text format).  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sippeerstatus' to 'Show the status of one or all of the sip peers.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'userevent' to 'Send an arbitrary event.  (Priv: user,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'listcategories' to 'List categories in configuration file.  (Priv: config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipshowregistrationsoutbound' to 'Lists PJSIP outbound registrations.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'skinnyshowline' to 'Show SKINNY line (text format).  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'status' to 'List channel status.  (Priv: system,call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sorcerymemorycachestaleobject' to 'Mark an object in a sorcery memory cache as stale.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'iaxpeers' to 'List IAX peers.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'ping' to 'Keepalive command.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'iaxnetstats' to 'Show IAX Netstats.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'login' to 'Login Manager.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queuereset' to 'Reset queue statistics.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipshowsubscriptionsinbound' to 'Lists subscriptions.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queuestatus' to 'Show queue status.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'donglerestart' to 'Restart a dongle. (Priv: system,config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'skinnyshowdevice' to 'Show SKINNY device (text format).  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queueremove' to 'Remove interface from queue.  (Priv: agent,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dahdidndoff' to 'Toggle DAHDI channel Do Not Disturb status OFF.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'unpausemonitor' to 'Unpause monitoring of a channel.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'playdtmf' to 'Play DTMF signal on a specific channel.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'redirect' to 'Redirect (transfer) a call.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dbdel' to 'Delete DB entry.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'donglesendpdu' to 'Send a PDU of message. (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipqualify' to 'Qualify a chan_pjsip endpoint.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'mailboxcount' to 'Check Mailbox Message Count.  (Priv: call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queuereload' to 'Reload a queue, queues, or any sub-section of a queue or queues.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipshowsubscriptionsoutbound' to 'Lists subscriptions.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queueadd' to 'Add interface to queue.  (Priv: agent,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dataget' to 'Retrieve the data api tree.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'donglereset' to 'Reset a dongle. (Priv: system,config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'agents' to 'Lists agents and their status.  (Priv: agent,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'filter' to 'Dynamically add filters for the current manager session.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sipnotify' to 'Send a SIP notify.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'faxsessions' to 'Lists active FAX sessions  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dbput' to 'Put DB entry.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dongleshowdevices' to 'List Dongle devices (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'moduleload' to 'Module management.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dahdihangup' to 'Hangup DAHDI Channel.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'devicestatelist' to 'List the current known device states.  (Priv: call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'iaxpeerlist' to 'List IAX Peers.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'parkinglots' to 'Get a list of parking lots  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgestartrecord' to 'Start recording a Confbridge conference.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'loggerrotate' to 'Reload and rotate the Asterisk logger.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipshowendpoint' to 'Detail listing of an endpoint and its objects.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'setvar' to 'Sets a channel variable or function value.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'absolutetimeout' to 'Set absolute timeout.  (Priv: system,call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'donglesendsms' to 'Send a SMS message. (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgesetsinglevideosrc' to 'Set a conference user as the single video source distributed to all other participants.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'localoptimizeaway' to 'Optimize away a local channel when possible.  (Priv: system,call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sorcerymemorycachepopulate' to 'Expire all objects from a memory cache and populate it with all objects from the backend.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'hangup' to 'Hangup channel.  (Priv: system,call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'prishowspans' to 'Show status of PRI spans.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pausemonitor' to 'Pause monitoring of a channel.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'skinnydevices' to 'List SKINNY devices (text format).  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'voicemailuserslist' to 'List All Voicemail User Information.  (Priv: call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'getvar' to 'Gets a channel variable or function value.  (Priv: call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgestoprecord' to 'Stop recording a Confbridge conference.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'confbridgelist' to 'List participants in a conference.  (Priv: reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'bridgedestroy' to 'Destroy a bridge.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipshowendpoints' to 'Lists PJSIP endpoints.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'stopmixmonitor' to 'Stop recording a call through MixMonitor, and free the recording's file handle.  (Priv: system,call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'corestatus' to 'Show PBX core status variables.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pridebugset' to 'Set PRI debug levels for a span  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'jabbersend' to 'Sends a message to a Jabber Client.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'donglestop' to 'Stop a dongle. (Priv: system,config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'cancelatxfer' to 'Cancel an attended transfer.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'createconfig' to 'Creates an empty file in the configuration directory.  (Priv: config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'updateconfig' to 'Update basic configuration.  (Priv: config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'originate' to 'Originate a call.  (Priv: originate,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipunregister' to 'Unregister an outbound registration.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sorcerymemorycachestale' to 'Marks ALL objects in a sorcery memory cache as stale.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dbdeltree' to 'Delete DB Tree.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'presencestatelist' to 'List the current known presence states.  (Priv: call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queuememberringinuse' to 'Set the ringinuse value for a queue member.  (Priv: agent,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'donglestart' to 'Start a dongle. (Priv: system,config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'changemonitor' to 'Change monitoring filename of a channel.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dbget' to 'Get DB Entry.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipnotify' to 'Send a NOTIFY to either an endpoint, an arbitrary URI, or inside a SIP dialog.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dahditransfer' to 'Transfer DAHDI Channel.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sippeers' to 'List SIP peers (text format).  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dahdidndon' to 'Toggle DAHDI channel Do Not Disturb status ON.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'bridgelist' to 'Get a list of bridges in the system.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'donglesendussd' to 'Send a ussd command to the dongle. (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'blindtransfer' to 'Blind transfer channel(s) to the given destination  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipregister' to 'Register an outbound registration.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pridebugfileunset' to 'Disables file output for PRI debug messages  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'bridgetechnologylist' to 'List available bridging technologies and their statuses.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'coresettings' to 'Show PBX core settings (version etc).  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'bridgeinfo' to 'Get information about a bridge.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'extensionstate' to 'Check Extension Status.  (Priv: call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queuerule' to 'Queue Rules.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'reload' to 'Send a reload event.  (Priv: system,config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipshowregistrationinboundcontactstatuses' to 'Lists ContactStatuses for PJSIP inbound registrations.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'faxsession' to 'Responds with a detailed description of a single FAX session  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'mailboxstatus' to 'Check mailbox.  (Priv: call,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dahdirestart' to 'Fully Restart DAHDI channels (terminates calls).  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sendtext' to 'Send text message to channel.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'voicemailrefresh' to 'Tell Asterisk to poll mailboxes for a change  (Priv: user,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'park' to 'Park a channel.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dialplanextensionadd' to 'Add an extension to the dialplan  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'faxstats' to 'Responds with fax statistics  (Priv: reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'modulecheck' to 'Check if module is loaded.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipshowresourcelists' to 'Displays settings for configured resource lists.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'getconfig' to 'Retrieve configuration.  (Priv: system,config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'mixmonitor' to 'Record a call and mix the audio during the recording.  Use of StopMixMonitor is required to guarantee the audio file is available for processing during dialplan execution.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'monitor' to 'Monitor a channel.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'atxfer' to 'Attended transfer.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'command' to 'Execute Asterisk CLI Command.  (Priv: command,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pjsipshowregistrationsinbound' to 'Lists PJSIP inbound registrations.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'listcommands' to 'List available manager commands.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sorcerymemorycacheexpireobject' to 'Expire (remove) an object from a sorcery memory cache.  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queuepause' to 'Makes a queue member temporarily unavailable.  (Priv: agent,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'queues' to 'Queues.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'sipqualifypeer' to 'Qualify SIP peers.  (Priv: system,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'dahdishowchannels' to 'Show status of DAHDI channels.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'parkedcalls' to 'List parked calls.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'challenge' to 'Generate Challenge for MD5 Auth.  (Priv: <none>)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'showdialplan' to 'Show dialplan contexts and extensions  (Priv: config,reporting,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'donglesetccwa' to 'Enable/Disabled Call-Waiting on a dongle. (Priv: config,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'pridebugfileset' to 'Set the file used for PRI debug message output  (Priv: system,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[Asterisk-Java ManagerConnection-0-Reader-0] WARN org.asteriskjava.manager.internal.ResponseBuilderImpl - Unable to set property 'stopmonitor' to 'Stop monitoring a channel.  (Priv: call,all)' on org.asteriskjava.manager.response.ManagerResponse: no setter. Please report at https://github.com/asterisk-java/asterisk-java/issues
[main] ERROR org.asteriskjava.pbx.internal.core.AsteriskPBX - java.lang.Exception: InitiateAction.AddExtentionFaileddialplan add extension asterisk-java-agi,1,AGI(agi://YYY.YYY.YYY.YYY/activityAgi), into asteriskjava
```

**Comment by seanbright** [comment]
What version of Asterisk is this?

**Comment by seanbright** [comment]
This may be a duplicate of #185
